Corona-Warn-App/src/main/res/values/styles.xml

```diff
@@ -257,6 +257,12 @@
         <item name="android:textColor">@color/colorTextPrimary1</item>
     </style>
 
+    <style name="subtitleBoldSixteen" parent="@style/TextAppearance.MaterialComponents.Subtitle1">
```

Comment:
Could you please clarify with the designers if we should introduce a new typographie?

Reply:
A new text style is wanted.

Reply:
Talked to the designers and it's indeed a new text style.